packages/integration-tests/suites/public-api/addBreadcrumb/template.hbs

```diff
@@ -1,11 +1,11 @@
-<!DOCTYPE html>
 <html>
   <head>
-    <meta charset="utf-8" />
-    <title></title>
-  <script src="{{htmlWebpackPlugin.options.initialization}}"></script>
+    {{#each htmlWebpackPlugin.options.sentry_bundles}}
+      <script src='{{this}}'></script>
+    {{/each}}
```

Comment:
I'm not sure this is going to work quite the way you're hoping. Every `tracing` bundle includes the full browser bundle, and in `generatePage.ts` below, you create tuples of [browser, tracing], meaning in the ES6 case, I think the ES5 version of the browser code in the tracing bundle would have the possibility to overwrite at least some of the ES6 browser code in the browser bundle.

The better solution here, IMHO, is for us to emit an ES6 tracing bundle, and then just use that, with no browser bundle. I've put a task on our board for this, and will do it in the next few days.

Reply:
Hmm I see, thanks for the explanation. 👍  So as I understand, to test browser-without-tracing bundles, we need to check whether a specific test requires tracing functionality (has imported `@sentry/tracing` in `init.js`), and use a bundle accordingly. Is that right? If so, I need to figure out how to detect what's imported in `init.js` at webpack build time.

Reply:
>  I need to figure out how to detect what's imported in init.js at webpack build time.

Yeah, unfortunately. And then test tracing-needing tests only against the tracing bundles and non-tracing-needing tests against all bundles, but separately rather than one tracing, one browser at the same time.